testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViewsLegacy.java

```diff
@@ -122,6 +122,20 @@ public void testArrayConstructionInView()
                 .hasMessageContaining("Function 'array' not registered");
     }
 
+    @Override
+    public void testMapConstructionInView()
```

Comment:
I did not test but does this mean legacy versions supported `DATE` and `WITH` cases? Because I don't see tests for them here.

Reply:
they do, because the HiveQL and Trino SQL syntax is the same for both.
For `WITH` specifically, see https://github.com/trinodb/trino/issues/5977#issuecomment-728310139